test/parsecBridge.js

```diff
@@ -42,6 +42,14 @@ contract('Parsec', (accounts) => {
         p[1] = await parsec.tipHash();
       });
 
+      it('should update slot instead of auction for same owner', async () => {
+        await parsec.bet(2, 10, alice, alice, {from: alice}).should.be.fulfilled;
+        await parsec.bet(2, 30, alice, alice, {from: alice}).should.be.fulfilled;
+        const slot = await parsec.slots(2);
+        assert.equal(Number(slot[1]), 30); // stake === 30
```

Comment:
shouldn't this be 10 + 30 = 40 ?

Reply:
because we pull all the tokens:
`token.transferFrom(msg.sender, this, _value);`
not only the diffirence:
`token.transferFrom(msg.sender, this, _value - slot.stake);`